redash/query_runner/__init__.py

```diff
@@ -177,3 +177,19 @@ def get_configuration_schema_for_query_runner_type(query_runner_type):
 def import_query_runners(query_runner_imports):
     for runner_import in query_runner_imports:
         __import__(runner_import)
+
+
+# for athena and presto
+def format_schema(results):
+    schema = {}
+    for row in results['rows']:
+        table_name = '{0}.{1}'.format(row['table_schema'], row['table_name'])
```

Comment:
No need to use `'{0}.{1}'` here instead of `'{}.{}'` since providing the order of the format string variables was only mandatory in Python 2.6 (and at least the Redash Docker container runs Python 2.7.x).